package.json

```diff
@@ -383,6 +383,12 @@
                 }
             ]
         },
+        "jsonValidation": [
+            {
+                "fileMatch": "staticwebapp.config.json",
```

Comment:
Eric has it like `/function.json` in the Functions repo, but I'm not really sure what the purpose/reason is.  @ejizba Is there any reason that we should fileMatch with a slash prefix?

Reply:
I don't remember. Only thing I can think of is maybe the slash prevents something like "test.function.json" from being affected - but I tested and doesn't seem to make a difference.